### DISCUSSION TITLE
Prolog 基础 <1> · /prolog-ji-chu-1

### DISCUSSION THREAD

**Discussion by draveness** [draveness/blog-comments#505] [Announcements]
https://draveness.me/prolog-ji-chu-1

**Comment by CHN-STUDENT** [comment]
最近在学prolog，感谢博主的优秀文章。

**Comment by Layljl0615** [comment]
在输入 brew cask install swi-prolog 之后 
出现 Error: Cask 'swi-prolog' is unavailable: No Cask with this name exists.
是什么原因
